spec/rubocop/cop/bundler/gem_version_spec.rb

```diff
@@ -52,6 +56,10 @@
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Gem version specification is forbidden.
         gem 'rubocop', '>= 1.5.0', '< 1.10.0', git: 'https://github.com/rubocop/rubocop'
         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Gem version specification is forbidden.
+        gem 'rubocop', github: 'rubocop/rubocop', tag: 'v1'
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Gem version specification is forbidden.
+        gem 'rubocop', git: 'https://github.com/rubocop/rubocop', tag: 'v1'
+        ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ Gem version specification is forbidden.
```

Comment:
These look strange. If `tag` or `ref` is specified with `git` or `github`, it's likely that a meaningful version of commit is being used. So, shouldn't these cases always be accepted?

Reply:
This is for `'EnforcedStyle' => 'forbidden'` which forbids adding a gem version (or in this case an equivalent git tag).

Reply:
Practically, if `:tag` or `:ref` with `:git` or `:github` is used, the pinned value should be respected. AFAIK, even `'EnforcedStyle' => 'forbidden'` should not be forbidden, as it is used for intentional forks and the like.

Reply:
changed